docs/detections/alerts-run-osquery.asciidoc

```diff
@@ -35,8 +36,18 @@ image::images/setup-query.png[width=80%][height=80%][Shows how to set up the que
 TIP: To save the query for future use, click *Save for later* and define the ID,
 description, and other {kibana-ref}/osquery.html#osquery-manage-query[details].
 
-. Review the results in the table. You can also navigate to *Discover* to dive deeper into the response,
-or use the drag-and-drop *Lens* editor to create visualizations.
+. Review the results in the table. You can also:
+** Navigate to *Discover* to dive deeper into the response.
+** Use the drag-and-drop *Lens* editor to create visualizations.
+** Click the *Timeline* button (image:images/timeline-button-osquery.png[Click markdown icon,20,20]) to investigate a single query result in Timeline or *Add to timeline investigation* to investigate all results.
++
+NOTE: The option to explore Osquery results in Timeline is only available from {elastic-sec}.
```

Comment:
There's no restriction, but the integration and lives queries are managed outside of Security so the barrier between Security and Kibana (which is where Osquery is loosely based) is kinda fuzzy. The UI for running queries from the Osquery feature in Kibana also looks pretty similar to the UI in Security, and offers all the same functionality (minus the Timeline feature ofc) so users might expect the same functionality in both? Anyways, if neither of these are significant enough to warrant the callout in the note, I'm ok removing it. The feature will also be mentioned the Security release notes only, so there's that as well (as far as making it clear that this is a Security-specific feature for Osquery).

Reply:
Good discussion here! I think it's safe to omit for now.

Reply:
Thanks for the extra context @nastasha-solomon! I see what you mean about the fuzzy border between Security and Kibana. I agree with @jmikell821, we could prob omit this note, since these docs assume a Security perspective, not the wider view of how Osquery works across the whole Elastic platform.